### PR TITLE
Pathway analysis training

### PR DESCRIPTION
**Pull request by kenza12**
Add new tutorial : **Evaluating and ranking a set of pathways based on multiple metrics**

The **video** was also submitted to the GTN : https://filesender.renater.fr/?s=download&token=31ced4e7-a55d-49dd-98e2-3a3634c6887e

Thank you.

### CODE REVIEW

File: topics/synthetic-biology/tutorials/pathway_analysis/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+We first perform an FBA (with COBRApy) optimizing the biomass reaction and record its maximal theoretical flux. The upper and lower bounds of the biomass reaction are then set to a same amount, equals to a fraction of its previously recorded optimum (default is 75% of its optimum). The method then performs a second FBA where biomass flux is enforced to this fraction of its optimum while optimizing the target production flux. Simulated fluxes are recorded directly into the SBML file and all changed flux bounds are reset to their original values before saving the output file.
+
+![Flux Balance Analysis](../../images/fba_calculation.png)
```
Comment (hexylena) [inline topics/synthetic-biology/tutorials/pathway_analysis/tutorial.md:105]: It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details.

Comment (kenza12) [reply topics/synthetic-biology/tutorials/pathway_analysis/tutorial.md]: Hi @hexylena , thanks for the review. I used the following syntax and description:

`This picture describes the process to obtain annotated SBML pathways with calculated fluxes. First, FBA tool takes as input one SBML representing the heterologous pathway and another SBML representing the chassis. The two SBMLs are merged to render an augmented model containing both the reactions of the heterologous pathway and the chassis. Then FBA tool uses the COBRApy package to optimize the producing flux of the target reaction, under the constraint that the flux of the biomass reaction should be equals to 75% of its maximal theoretical value. At the end, the calculated fluxes are recorded as annotations into the SBML of the heterologous pathway.`

Is this ok for you ?

Comment (hexylena) [reply topics/synthetic-biology/tutorials/pathway_analysis/tutorial.md]: Yes, absolutely! That's great, that's exactly what folks who cannot see the pictures need, to be able to understand the content. Could I ask you to do something similar for the other images you use? Then it's perfect and we can merge :)

File: topics/synthetic-biology/tutorials/pathway_analysis/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+<img src="../../images/fba_calculation.png" alt="This picture describes the process to obtain annotated SBML pathways with calculated fluxes. First, FBA tool takes as input one SBML representing the heterologous pathway and another SBML representing the chassis. The two SBMLs are merged to render an augmented model containing both the reactions of the heterologous pathway and the chassis. Then FBA tool uses the COBRApy package to optimize the producing flux of the target reaction, under the constraint that the flux of the biomass reaction should be equals to 75% of its maximal theoretical value. At the end, the calculated fluxes are recorded as annotations into the SBML of the heterologous pathway." loading="lazy">
```
Comment (hexylena) [inline topics/synthetic-biology/tutorials/pathway_analysis/tutorial.md:105]:
```suggestion
![This picture describes the process to obtain annotated SBML pathways with calculated fluxes. First, FBA tool takes as input one SBML representing the heterologous pathway and another SBML representing the chassis. The two SBMLs are merged to render an augmented model containing both the reactions of the heterologous pathway and the chassis. Then FBA tool uses the COBRApy package to optimize the producing flux of the target reaction, under the constraint that the flux of the biomass reaction should be equals to 75% of its maximal theoretical value. At the end, the calculated fluxes are recorded as annotations into the SBML of the heterologous pathway.](../../images/fba_calculation.png)
```

File: topics/synthetic-biology/tutorials/pathway_analysis/tutorial.md (hunk 1 above)
Comment (kenza12) [reply topics/synthetic-biology/tutorials/pathway_analysis/tutorial.md]: Ok great, thank you. I just added the other descriptions.

I also posted a video like I mentioned at the beginning: https://filesender.renater.fr/?s=download&token=31ced4e7-a55d-49dd-98e2-3a3634c6887e , submitted to GTN. I don't know if I have to correct the captions ?

Comment (shiltemann) [reply topics/synthetic-biology/tutorials/pathway_analysis/tutorial.md]: Thanks for the great alt text @kenza12! yes, I will process your video in the video library, and send you captions to correct in the next couple of days

**Review by hexylena — APPROVED** [review]
Looks fantastic @kenza12 !!

**Comment by kenza12** [comment]
> Looks fantastic @kenza12 !!

Great, thank you @hexylena !!

**Comment by shiltemann** [comment]
Here are the captions @kenza12: https://docs.google.com/document/d/1GrtFMvoQLMCNxIMc501_ECvQudaPH2cwEs9Zy2Ee4BU/edit?usp=sharing

Thanks!

**Comment by shiltemann** [comment]
perfect, thanks! The video is now live on the GTN: https://training.galaxyproject.org/training-material/topics/synthetic-biology/

**Pull request merged**